### ISSUE TITLE
What is the expected "global batch size"?

### ISSUE THREAD

**Issue by ohmeow** [huggingface/alignment-handbook#50]
In the recipes README there is this statement:

>  If you scale up/down the number of GPUs, we recommend also scaling up the per-device batch size or number of gradient accumulation steps to keep the global batch size constant (and thus replicate our results).

Q: What is the expected "global batch size"?

For example, I'm trying to run this on 2x3090s  and need to know what the expected global batch size is so I can adjust the accumulation steps and per device train batch size.

Thanks much!

**Comment by timothylimyl** [comment]
@ohmeow 

For SFT, it's 512.

For DPO, it's 32.
